pkg/kapp/logs/pod_container_log.go

```diff
@@ -144,10 +145,13 @@ func (l PodContainerLog) obtainStream(ui ui.UI, linePrefix string, cancelCh chan
 
 		if !isWaitingMsgPrintedOnce {
 			ui.BeginLinef("%s# waiting for '%s' logs to become available...\n", linePrefix, l.tag)
+			isWaitingMsgPrintedOnce = true
 			if !l.opts.Follow {
+				if err == nil {
+					err = errors.New(fmt.Sprintf("Container %s not in Ready state", l.container))
```

Comment:
We stick to using `fmt.Errorf` I guess
(I think the linter is pointing out the same)

Reply:
Adopted